### PR TITLE
Upgrade to go v1.13

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

**Comment by dolmen** [comment]
Why?

**Comment by chenrui333** [comment]
@dolmen, why not?

**Comment by niemeyer** [comment]
We want to make sure it continues to work in prior versions. Adding to the list is fine, though.

**Comment by laverya** [comment]
We should probably also compile with `1.10`-`1.12`, just for completeness.

**Comment by chenrui333** [comment]
That makes sense, I will add more golang versions for testing.

**Comment by chenrui333** [comment]
@niemeyer I think it should be good now!

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
+  - "1.9.x"
+  - "1.10.x"
+  - "1.11.x"
+  - "1.12.x"
+  - "1.13.x"
+  - tip
+
+os:
+  - linux
+  - osx
```
Comment (niemeyer) [inline .travis.yml]: Can we please restrict this PR to this content only. The rest is unrelated to what the PR proposes and I'd rather get the main intention in right now than keep discussing unrelated bits here.

**Review by niemeyer — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (chenrui333) [reply .travis.yml]: Sounds good, I will refine the travis build changes.

Comment (chenrui333) [reply .travis.yml]: @niemeyer Updated, now the syntax should be using the latest versions of each golang lines.

Comment (laverya) [reply .travis.yml]: @chenrui333 I think what was meant by that was to not include the go.mod/go.sum changes in this PR

Comment (chenrui333) [reply .travis.yml]: Interesting, so we just need travis build config changes?

Comment (laverya) [reply .travis.yml]: Presumably, since the pr into the v3 branch I made doesn't touch the go.mod/go.sum files, and they're the same between the branches

**Comment by niemeyer** [comment]
I went ahead and did the change simply adding the missing versions on 1ed59511.

**Comment by chenrui333** [comment]
Thanks @niemeyer.

Also, I just create a new PR for cleanup `go.mod` and `go.sum`.

**Pull request closed without merging**